code/game/gamemodes/scoreboard.dm

```diff
@@ -2,6 +2,19 @@
 
 	mode.declare_completion()
 	completions += "[mode.dat]<HR>"
+	
+	completions += "<h2><font color='red'>Syndicate</font> Specials</h2>"
+	if(syndicate_code_phrase)
+		completions += "<BR>The Syndicate code phrases were:<BR>"
+		completions += "<font color='red'>[syndicate_code_phrase.Join(", ")]</font>"
+	if(syndicate_code_response)
+		completions += "<BR>The Syndicate code responses were:<BR>"
+		completions += "<font color='red'>[syndicate_code_response.Join(", ")]</font>"
+	
+	if(score["implant_phrases"].len)
```

Comment:
DreamChecker is complaining about this line, because there's no guarantee at all that this thing will be a list. You want to typecast it first.